### DISCUSSION TITLE
Can't run the examples

### DISCUSSION THREAD

**Discussion by shantnu** [ephread/inkgd#54] [General]
Not sure what Im doing wrong-- I cloned the repo, and tried to run examples/scenes/main.tscn. But I get this error:

```

E 0:00:20.511   call: 'ink_file' is null, did Godot import the resource correctly?
  <C++ Source>  modules/gdscript/gdscript_functions.cpp:774 @ call()
  <Stack Trace> ink_player.gd:750 @ _push_error()
                ink_player.gd:269 @ create_story()
                story_player.gd:86 @ _ready()
```

Using Godot 3.4.2

**Comment by ephread** [comment]
Hey @shantnu, I just tried with a fresh clone. On first launch, inkgd couldn't be initialised (as expected because Godot doesn't ensure that all resources are reimported before activating the plugins), but after reloading the project, everything went well.

I'm not sure what could be going on, are you certain you launched `main.tscn` and not `story_player.tscn`? You could also try the [prepackaged example](https://github.com/ephread/inkgd/releases/download/0.4.3/inkgd-example-0.4.3.zip) that already contains a `.import` folder.

**Reply by shantnu** [reply]
Sorry, just saw this, didnt get the email notification

Thanks, it is working with the prepackaged example 👍
